targets/unity-v2/make.js

```diff
@@ -378,7 +418,14 @@ function getAuthParams(apiCall) {
     return "AuthType.None";
 }
 
-function getRequestActions(tabbing, apiCall) {
+function getInstanceParams(apiCall) {
+    if (apiCall.auth === "SecretKey")
+        return "authenticationContext, apiSettings"
```

Comment:
@Vicodex This condition seems VERY weird to me.
Why is this the case?
@melihceyhan, Feel free to answer or ignore. I just want to know why this choice is made.

Reply:
we use authenticationContext in MakeApiCall for only ServerInstance so we do not need to pass it. By the way apiSettings is needed for all Instance-able ones. I can also pass authenticationContext for all instance-able ones if you want. We should also wait for @Vicodex's comment

Reply:
If we can pass both arguments in call cases i think it will make it less confusing. Even if authenticationContext is not used in one of the cases, it looks like it won't harm if we pass it?

---
In reply to: [261843518](https://github.com/PlayFab/SDKGenerator/pull/420#discussion_r261843518) [](ancestors = 261843518)

Reply:
Yes i think so. It wont harm anything. I commited it. Is there anything more to correct for this task?